docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc

```diff
@@ -80,6 +91,8 @@ We recommend only having one fleet-server policy.
 If this is the default policy for fleet-server,
 it is picked automatically by fleet-server for enrollment.
 
+NOTE: Replace `docker.elastic.co/beats/elastic-agent` with `docker.elastic.co/beats/elastic-agent-complete` and use the `elastic-agent` user instead of root to run Synthetics Browser tests. Synthetics tests cannot run under the root user. See {observability-guide}/synthetics-quickstart-fleet[Synthetics Fleet Quickstart] for more information.
```

Comment:
I ran this command connecting to a local instance using `elastic-package` and connecting to cloud instance, without changing the user and it worked, so it's not using root by default. We may be able to remove the extra note regarding the user, but may want also may elect to keep it for added clarity. What do you think @andrewvc